.gitignore

```diff
@@ -7,3 +7,4 @@ build
 def.sdmx.json
 vendor/
 .go/
+dp-dataset-api
```

Comment:
newlline needed at end of file

Reply:
thank you. is this something we should lint?